src/components/cards/project-card/index.tsx

```diff
@@ -26,11 +27,11 @@ const ProjectCard: FC<ProjectCardProps> = ({
     <S.Card {...rest}>
       <S.Header>
         <S.Cover
-          url={cover}
+          url={getResizedImgUrl(cover, 758)} // why is this number here (758), it is the largest width I was able to find with all window sizes - and since I do not want to repeat the CSS logic here, I took the largest
           aria-label={`${t('cards.project.coverAriaLabel')} ${title}`}
         />
         <S.Logo
-          url={logo}
+          url={getResizedImgUrl(logo, 82)}
```

Comment:
Navrhoval bych na ta loga spíš nesahat – tam nečekám, že budou nějak obrovská, a zároveň bývají třeba pečlivěji optimalizovaná, takže další zmenšování bude na škodu. Co myslíš?

Reply:
Souhlas, klidně, nechme to jen u těch fotek, předělám večer. I opravím ty nully v event.coverUrl.